serial_views.py

```diff
@@ -86,6 +174,107 @@ def district_feedin_series(self):
         pass
 
 
+# ###########TESTING#####################
+# sa = Serializer()
+# POST_REGION = 'Bayern'
+# sa.ppr_view(request=POST_REGION)
+# print('WAIT')
+# ########################################
```

Comment:
You can always leave this changes out by using `git stash` before you do commits (and then `git stash apply` to get those local changes back)

Reply:
thats true :) will try to remember next time